recipes/caelus/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "caelus" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sayerhs/cpl/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1bd247675b7c13b022afcee3f17e24d7080bc2f6b8e9b090492726c7bb0a7e6f
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - caelus = caelus.scripts.caelus:main
+    - caelus_tutorials = caelus.scripts.caelus_tutorials:main
+    - caelus_sim = caelus.scripts.caelus_sim:main
+
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+
+  run:
+    - python >=3.6
+    - six
+    - numpy
+    - matplotlib
+    - pyyaml
+    - ply
+    - pytz
+    - jinja2
+
+test:
```

Add a check for requirements consistency.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```